components/frontend/src/source/Source.js

```diff
@@ -88,21 +89,47 @@ function Parameters({ metric, source, source_uuid, connection_error, parse_error
             </Grid.Row>
             {connection_error && <ErrorMessage title="Connection error" message={connection_error} />}
             {parse_error && <ErrorMessage title="Parse error" message={parse_error} />}
+            {config_error && <ErrorMessage title="Configuration error" message={config_error} formatAsText={true} />}
         </Grid>
     )
 }
 
 export function Source({ metric, source_uuid, first_source, last_source, measurement_source, report, changed_fields, reload }) {
+    const dataModel = useContext(DataModel)
     const source = metric.sources[source_uuid];
+    const sourceName = get_source_name(source, dataModel);
+    const metricName = get_metric_name(metric, dataModel);
     const connectionError = measurement_source?.connection_error || "";
     const parseError = measurement_source?.parse_error || "";
-    const configurationTabLabel = connectionError || parseError ? <Label color='red'>{"Configuration"}</Label> : "Configuration";
+    const referenceManualURL = `https://quality-time.readthedocs.io/en/v${process.env.REACT_APP_VERSION}/reference.html`;
+    const configErrorMessage = <>
+        <p>
+            {sourceName} cannot be used to measure {metricName}. This configuration error occurs if the type of a
+            metric is changed to a metric type that is not supported by the configured source type(s).
+        </p>
+        <p>
+            There are several ways to remediate this error:
```

Comment:
Perhaps use "remedy" as it is also correct, but simpler

Reply:
Or "fix", even easier :-)